Steps4Impact/DashboardV2/DashboardViewController.swift

```diff
@@ -62,6 +64,13 @@ class DashboardViewController: TableViewController {
     }
   }
 
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    // Asking for push notification permission after login
+    AppController.shared.askForPushNotificationPermissions()
```

Comment:
lets not ask for notifications as soon as dashboard is loaded. rather lets put up an alert asking if the user wants notifications then ask for notification permissions in viewdidappear. Lets also make sure we only ask once.